index.html

```diff
@@ -111,13 +112,13 @@
   When <dfn>toJSON</dfn> is called, run [[!WEBIDL]]'s <a data-cite="WEBIDL#default-tojson-operation">default toJSON operation</a>.
 
   ### <dfn>name</dfn> attribute
-  The <a>name</a> attribute MUST return the server-specified metric-name.
+  The <a>name</a> attribute MUST return the server-specified metric name.
 
   ### <dfn>duration</dfn> attribute
-  The <a>duration</a> attribute MUST return a double that contains the server-specified metric-value, or value `0.0`.
+  The <a>duration</a> attribute MUST return a double that contains the server-specified metric value, or value `0.0`.
```

Comment:
Can you map this to the `server-timing-param-value` when `server-timing-param-name==duration`?